ui/components/multichain/token-list/token-list.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import classnames from 'classnames';
+import {
+  DISPLAY,
+  FLEX_DIRECTION,
+  FONT_WEIGHT,
+  JustifyContent,
+  Size,
+  TextColor,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import {
+  AvatarNetwork,
+  AvatarToken,
+  AvatarWithBadge,
+  AVATAR_WITH_BADGE_POSTIONS,
+  Text,
+} from '../../component-library';
+import Box from '../../ui/box/box';
+import { getNativeCurrencyImage } from '../../../selectors';
+
+export const NewTokenList = ({
+  className,
+  onClick,
+  tokenSymbol,
+  tokenImage,
+  primary,
+  secondary,
+}) => {
+  const primaryTokenImage = useSelector(getNativeCurrencyImage);
+  return (
+    <Box
+      className={classnames('token-list', className)}
+      display={DISPLAY.FLEX}
+      flexDirection={FLEX_DIRECTION.COLUMN}
+      gap={4}
+    >
+      <Box
+        className="token-list__container-cell"
+        display={DISPLAY.FLEX}
+        flexDirection={FLEX_DIRECTION.ROW}
+        padding={4}
+        onClick={onClick}
+        role="button"
+        tabIndex={0}
+        onKeyPress={(event) => {
+          if (event.key === 'Enter') {
+            onClick();
+          }
+        }}
+      >
+        <AvatarWithBadge
+          badgePosition={AVATAR_WITH_BADGE_POSTIONS.TOP}
+          badge={
+            <AvatarNetwork
+              size={Size.XS}
+              name="Ethereum"
```

Comment:
I think we should use the `networkNameEthereum` localization.

Reply:
Done